forecastio.class.nut

```diff
@@ -53,16 +54,17 @@ class Forecastio {
             }
         }
 
-        local url = FORECAST_URL + _apikey + "/" + format("%.6f", latitude) + "," + format("%.6f", longitude);
+        local url = FORECAST_URL + _apikey + "/" + format("%.6f", latitude) + "," + format("%.6f", longitude) + "?units=" + units;
         return _sendRequest(http.get(url), callback);
     }
 
-    function timeMachineRequest(longitude = 999, latitude = 999, time = null, callback = null) {
+    function timeMachineRequest(longitude = 999, latitude = 999, time = null, units = us, callback = null) {
         // Parameters:
         //  1. Longitude of location for which a forecast is required
         //  2. Latitude of location for which a forecast is required
         //  3. A Unix time or ISO 1601-formatted string
-        //  4. Optional synchronous operation callback
+        //  4. Optional units parameter which defaults to us is not specified
```

Comment:
Should be:
//  4. Optional units parameter which defaults to "us" if not specified